src/Commands/Import/ImportDatabaseCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Import;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
```

Comment:
All three of your Import commands use this SiteAwareInterface/Trait. Why not pull this out into an abstract ImportBaseCommand and have all three extend it in order to DRY this up?